cmds/losetup/losetup.go

```diff
@@ -15,90 +15,57 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
 	"os"
-	"syscall"
-)
 
-const (
-	/*
-	 * IOCTL commands --- we will commandeer 0x4C ('L')
-	 */
-	LOOP_SET_CAPACITY = 0x4C07
-	LOOP_CHANGE_FD    = 0x4C06
-	LOOP_GET_STATUS64 = 0x4C05
-	LOOP_SET_STATUS64 = 0x4C04
-	LOOP_GET_STATUS   = 0x4C03
-	LOOP_SET_STATUS   = 0x4C02
-	LOOP_CLR_FD       = 0x4C01
-	LOOP_SET_FD       = 0x4C00
-	LO_NAME_SIZE      = 64
-	LO_KEY_SIZE       = 32
-	/* /dev/loop-control interface */
-	LOOP_CTL_ADD      = 0x4C80
-	LOOP_CTL_REMOVE   = 0x4C81
-	LOOP_CTL_GET_FREE = 0x4C82
-	SYS_ioctl         = 16
+	. "github.com/u-root/u-root/pkg/losetup"
```

Comment:
Please don't use . imports -- they are reserved for some special cases. (I've never actually seen them used for a good reason.)